proto/mls/message_contents/content.proto

```diff
@@ -91,14 +91,5 @@ message MessageHistoryReply {
   // Where the messages can be retrieved from
   string url = 2;
   // Generated input 'secret' for the AES Key used to encrypt the message-bundle
-  MessageHistoryKeyType encryption_key = 3;
-  // Generated input 'secret' for the HMAC Key used to sign the bundle_hash
-  MessageHistoryKeyType signing_key = 4;
-  // HMAC Signature of the message-bundle
-  bytes bundle_hash = 5;
-}
-
-// Key used to encrypt or sign the message-bundle
-message MessageHistoryKeyType {
-  oneof key { bytes chacha20_poly1305 = 1; }
```

Comment:
This does have value in at least telling you what type of key it is, and allowing us to change key types later.